src/library-authoring/import-course/stepper/ReviewImportDetails.test.tsx

```diff
@@ -100,22 +122,48 @@ describe('ReviewImportDetails', () => {
 
   it('renders warning when unsupportedBlockPercentage > 0', async () => {
     (useCourseDetails as jest.Mock).mockReturnValue({ isPending: false, data: { title: 'Test Course' } });
+    (useLibraryBlockLimits as jest.Mock).mockReturnValue({
+      isPending: false,
+      data: { maxBlocksPerContentLibrary: 100 },
+    });
     (useMigrationInfo as jest.Mock).mockReturnValue({
       isPending: false,
       data: null,
     });
-    (useGetBlockTypes as jest.Mock).mockReturnValueOnce({
-      isPending: false,
-      data: {
-        chapter: 1,
-        sequential: 2,
-        vertical: 3,
-        'problem-builder': 1,
-        html: 1,
-      },
+    (useGetBlockTypes as jest.Mock).mockImplementation((args) => {
+      // Block types query for children of unsupported blocks
```

Comment:
I updated this mock in this way because an issue that renders the component twice